runtime/app.go

```diff
@@ -86,7 +85,6 @@ func (a *App) Load(loadLatest bool) error {
 
 	if len(a.config.InitGenesis) != 0 {
 		a.ModuleManager.SetOrderInitGenesis(a.config.InitGenesis...)
-		a.SetInitChainer(a.InitChainer)
```

Comment:
Isn't this needed?

Reply:
I guess we need to set the module version map. But IMHO, this should somehow be done automatically with app config

Reply:
`app_v2.go` already sets it, so this was overwritting it and not calling `	app.UpgradeKeeper.SetModuleVersionMap(ctx, app.ModuleManager.GetVersionMap())`.
Didn't run the integration tests locally, if you use runtime standalone it is indeed needed.

Reply:
I see, let me check 👍🏾

Reply:
Because we need the store to be available, I have not found a clean way.